sql-cli/sql_cli/connections.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import yaml
+from airflow.api_connexion.schemas.connection_schema import connection_schema
+from airflow.models import Connection
+from airflow.utils.session import create_session
+
+from sql_cli.settings import SQL_CLI_PROJECT_DIRECTORY
+
+
+def _load_yaml_connections(environment: str) -> list[dict[str, Any]]:
+    """Gets the configuration yaml for the given environment and loads the connections from it into a dictionary"""
+    config_file = SQL_CLI_PROJECT_DIRECTORY / "config" / environment / "configuration.yaml"
+    if not config_file.exists():
+        raise FileNotFoundError(
+            f"Config file configuration.yaml does not exist for environment {environment}"
+        )
+
+    with open(config_file) as connections_file:
+        connections: list[dict[str, Any]] = yaml.safe_load(connections_file)["connections"]
```

Comment:
@pankajkoti we also should expand environment variables, similar to:
```
https://github.com/astronomer/astro-sdk/blob/main/python-sdk/conftest.py#L48
```
If the users don't want to hard-code the password and other sensitive data inside the YAML file itself.

It would be great if you could also have a test for this.

Reply:
Thanks, this was needed. Using a redshift connection now with secrets loaded for environment variable. Altered the test to check that Redshift connections passes locally with .env file environment vars and in CI using CI yaml induced env variables for the Redshift in the Tests job.